app/controllers/support_interface/manage_vendors_controller.rb

```diff
@@ -0,0 +1,12 @@
+module SupportInterface
+  class ManageVendorsController < SupportInterfaceController
+    def index
+      @providers = Provider.all
+    end
+
+    def create
+      GenerateVendorProviders.call
+      redirect_back(fallback_location: root_path)
```

Comment:
This seems a bit weird, could it be just redirect_to the index? Minor

Reply:
Do you mean redirect to index on fallback?